nixos/doc/manual/development/importing-modules.xml

```diff
@@ -0,0 +1,82 @@
+<section xmlns="http://docbook.org/ns/docbook"
+         xmlns:xlink="http://www.w3.org/1999/xlink"
+         xmlns:xi="http://www.w3.org/2001/XInclude"
+         version="5.0"
+         xml:id="sec-importing-modules">
+
+<title>Importing Modules</title>
+
+<para>Sometimes NixOS modules exist outside of Nixpkgs, and you want
+to get them into your NixOS configuration.
+</para>
+
+<para>
+The simplest way to do this is to import the module:
+</para>
+
+<programlisting>
+{ config, lib, pkgs, ... }:
+
+{
+  imports =
+    [ # Use a locally-available module definition in
+      # ./example-module/default.nix
+        ./example-module
+    ];
+
+  services.exampleModule.enable = true;
+}
+</programlisting>
+
+<para>
+If the list of modules you need to import grows large, you can move
```

Comment:
Whether the list grows large or not is irrelevant; it's always possible to keep it outside. I would drop this example as it doesn't add anything.